rasa/core/nlg/callback.py

```diff
@@ -108,3 +114,25 @@ def validate_response(content: Optional[Dict[Text, Any]]) -> bool:
                 f"`nlg_response_format_spec` function from this same module: "
                 f"https://github.com/RasaHQ/rasa/blob/main/rasa/core/nlg/callback.py"
             )
+
+    @staticmethod
+    def fetch_response_id(
+        utter_action: Text,
+        tracker: DialogueStateTracker,
+        output_channel: Text,
+        domain_responses: Optional[Dict[Text, List[Dict[Text, Any]]]],
+    ) -> Text:
```

Comment:
Can you please add a small docstring or a comment here?

Reply:
Addressed: https://github.com/RasaHQ/rasa/pull/12575/commits/27e98159b985b91c4a2204fc18cc089e54d75efb